image/static-bin/import-additional-cas

```diff
@@ -2,4 +2,14 @@
 
 set -euo pipefail
 
-cp -L /usr/local/share/ca-certificates/* /etc/pki/ca-trust/source/anchors && update-ca-trust || echo "No custom certificates"
+copy_existing () {
+    src=$1
+    if [ -d "$src" ] && [ "$(ls -A -I "..*" "$src")" ]; then
```

Comment:
Previously we log info if no certs were copied I think this could be beneficial to debug issues